apidoc/Titanium/UI/Color.yml

```diff
@@ -9,18 +9,25 @@ description: |
     <Titanium.UI.fetchSemanticColor> method, which will return an instance to be passed to UI components.
     This instance will automatically "adapt" the receiver to light/dark mode changes.
 
+    The only current instance you'd receive this type on Android is through the <Titanium.Android.UI.getColorResource> method.
     Note that Android does not currently support passing in instances of this type for UI components/properties!
+    You must convert to a hex string representation to pass this in to any color properties on Android.
 
     See [UI Element Colors](https://developer.apple.com/documentation/uikit/uicolor/ui_element_colors?language=objc) and
     [Standard Colors](https://developer.apple.com/documentation/uikit/uicolor/standard_colors?language=objc) for lists of
     named colors to pass to <Titanium.UI.fetchSemanticColor> on iOS.
+
+    See [R.color](https://developer.android.com/reference/android/R.color) for a list of system color names to use to pass to
+    <Titanium.Android.UI.getColorResource> on Android.
 extends: Titanium.Proxy
 since: "9.1.0"
-platforms: [iphone, ipad]
+platforms: [iphone, ipad, android]
 methods:
   - name: toHex
-    summary: Returns a hex string representation of the color
+    summary: Returns a hex string representation of the color (in the RRGGBB or AARRGGBB formats)
```

Comment:
I can see that our `TiColorHelper.parseColor()` method has a `shortHexPattern` which accepts a `#AARRGGBB` or `#RRGGBB` formatted string. Is this why you switched it over to ARGB from RGBA? (I'm curious what iOS supports now.)

Reply:
Yes, I originally was trying to use RGBA, but then found out that Android natively uses ARGB, and that iOS code we already had used ARGB as well.

So there's clearly a mismatch with CSS here, but I did not want to break existing code.

Reply:
At first, I saw that iOS returned RGBA in TiUtils.hexColorValue - so I aimed for RGBA. But then I noticed under the hood the color parsing code actually expected ARGB hex values: https://github.com/appcelerator/titanium_mobile/blob/master/iphone/TitaniumKit/TitaniumKit/Sources/API/Webcolor.m#L163-L198

At that point since iOS code already had expected ARGB inputs, and Android natively uses ARGB (and expected it in TiColorHelper.parseCOlor, vbecaus eat defers to Color.parseCOlor which expects ARGB) - I opted to just line it all up as ARGB. To move over to RGBA makes sense to align with CSS (and was why I approached it first), but would be a breaking change on both platforms.

Reply:
Okay. Makes sense. 👍